src/test/java/com/lgtm/easymoney/services/impl/RequestServiceImplTest.java

```diff
@@ -0,0 +1,145 @@
+package com.lgtm.easymoney.services.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+
+import com.lgtm.easymoney.enums.Category;
+import com.lgtm.easymoney.enums.TransactionStatus;
+import com.lgtm.easymoney.exceptions.InvalidUpdateException;
+import com.lgtm.easymoney.exceptions.ResourceNotFoundException;
+import com.lgtm.easymoney.models.Transaction;
+import com.lgtm.easymoney.models.User;
+import com.lgtm.easymoney.payload.RequestReq;
+import com.lgtm.easymoney.payload.RequestRsp;
+import com.lgtm.easymoney.payload.ResourceCreatedRsp;
+import com.lgtm.easymoney.payload.TransactionRsp;
+import com.lgtm.easymoney.payload.TransferReq;
+import com.lgtm.easymoney.payload.TransferRsp;
+import com.lgtm.easymoney.services.TransactionService;
+import com.lgtm.easymoney.services.UserService;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.test.context.junit4.SpringRunner;
+
+/**
+ * Unit test for RequestService.
+ * {@link com.lgtm.easymoney.services.impl.RequestServiceImpl}
+ * */
+@RunWith(SpringRunner.class)
+public class RequestServiceImplTest {
+  @Mock
+  private TransactionService transactionService;
+  @Mock
+  private UserService userService;
+  @InjectMocks
+  private TransferServiceImpl transferService;
+  @InjectMocks
+  private RequestServiceImpl requestService;
+  private TransferReq transferReq;
+  private RequestReq requestReq;
+  private User user1;
+  private User user2;
+  private Transaction transaction;
+  private Long id1 = 1L;
+  private Long id2 = 2L;
+  private BigDecimal amount = new BigDecimal(10);
+  private Category category = Category.PARTY;
+  private String description = "test transfer";
+  private Long transactionId = 11L;
+  private BigDecimal user1Balance = new BigDecimal(50);
+  private BigDecimal user2Balance = new BigDecimal(100);
+  private Date lastUpdateTime = new Date(20221020L);
+  private String message = "Retrieved user's requests!";
+
+  /**
+   * Set up reusable test fixtures.
+   * */
+  @Before
+  public void setUp() {
+    // transferReq
```

Comment:
to reviewers: will remove comments and cleanup shit in next PR. this is a base PR